core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -22,55 +25,62 @@ import scala.util.Try
  */
 class HttpExecutionServiceDAO( executionServiceURL: String, submissionTimeout: FiniteDuration )( implicit val system: ActorSystem ) extends ExecutionServiceDAO with DsdeHttpDAO with Retry with FutureSupport with LazyLogging {
 
+  //sendReceive, but with gzip.
+  //Two versions, one of which overrides the timeout.
```

Comment:
1 + 2: yes, ok.
3: I was trying to avoid redefining the default timeout. Copying it out from Spray internals seemed gross to me.